src/pages/Games/SprintGame/styles.module.scss

```diff
@@ -0,0 +1,12 @@
+.card {
+  border: 1px solid #000000;
+  padding: 1rem;
+}
+
+.card__word {
```

Comment:
Empty style block

Reply:
> Empty style block

As it was mentioned, styles will be written after functional implementation.